mis_builder_report_xlsx/wizards/project_mis_export_wizard.py

```diff
@@ -0,0 +1,21 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, _, api
+
+
+class ProjectProject(models.TransientModel):

```

Comment:
Better this by convention:
```suggestion
class ProjectMisExportWizard(models.TransientModel):

```